lib/permissions/src/schema/schema.zed

```diff
@@ -2,5 +2,6 @@ definition user {}
 
 definition workspace {
     relation approver: user
+    relation owner: user
     permission approve = approver
```

Comment:
Did you want `permission approve = approver+owner` as well, like in `component/spicedb/schema.zed`?